src/Classes/Canvas.ts

```diff
@@ -5,19 +5,24 @@ import {Container} from "./Container";
 import {PolygonMaskType} from "../Enums/PolygonMaskType";
 import {IDimension} from "../Interfaces/IDimension";
 import {Utils} from "./Utils/Utils";
+import {IRgb} from "../Interfaces/IRgb";
+import {Particle} from "./Particle";
+import {ICoordinates} from "../Interfaces/ICoordinates";
+import {CanvasUtils} from "./Utils/CanvasUtils";
 
 /**
  * Canvas manager
  */
 export class Canvas {
     /**
-     * The particles canvas
+     * The particles canvas context
      */
-    public element: HTMLCanvasElement;
+    private readonly context: CanvasRenderingContext2D | null;
+
     /**
-     * The particles canvas context
+     * The particles canvas
      */
-    public context: CanvasRenderingContext2D | null;
+    public element: HTMLCanvasElement;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance field not allowed after declaration of private instance field. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5171185)